### DISCUSSION TITLE
Icon to toggle, card for more-info, possible?

### DISCUSSION THREAD

**Discussion by johannes-z** [piitaya/lovelace-mushroom#711] [Q&A]
Is there a way to toggle an entity when the icon is clicked, but bring up the more-info dialog when I click the card (not the icon within)?

**Comment by piitaya** [comment]
Apple Homekit do this I think. I think it can be good to have this behavior for all cards to keep consistency 🙂

**Reply by johannes-z** [reply]
Yeah I saw the latest video of [*Smart Home Solver*](https://youtu.be/EuyTN63pMQE?t=33) where he showed the new homekit UI. I like the idea of two different kinds of actions with a single press, instead of long press.

Is this something you're interested in for mushroom UI? If so I might take a look, doesn't look too hard to add.

**Reply by mwolter805** [reply]
Would be great to separate the actions (Tap, Hold and Double Tap) for the icon and the rest of the card (Primary information, Secondary information, etc.) on all cards. This would help mushroom cards match the functionally of the default tile card as well as apple home and others.

Thank you!
